docs/volcano_integration.md

```diff
@@ -0,0 +1,161 @@
+# Integration with Volcano for Batch Scheduling
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workloads.
+With the integration with Volcano, Flink job and task managers can be scheduled simultaneously, which is particularly suitable for 
+clusters with resource shortage.
+
+## Prerequisites
+
+## Install Volcano
+
+- Install from provided demo
+
+Run the following 
+```bash
+kubectl apply -f https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/volcano-development.yaml
+```
+
+- Install with advanced settings
+
+Please refer to [Volcano Official Guide](https://volcano.sh/docs/getting-started/)
+
+### Verify Volcano is up and running
+
+```bash
+$ kubectl get pod -n volcano-system
+NAME                                      READY   STATUS      RESTARTS   AGE
+pod/volcano-admission-75688c79bf-b8fmj    1/1     Running     0          52s
+pod/volcano-admission-init-d684j          0/1     Completed   0          53s
+pod/volcano-controllers-d87bdbd7c-q6ds6   1/1     Running     0          52s
+pod/volcano-scheduler-5476779fd9-8rslv    1/1     Running     0          52s
+
+```
+ 
+## Install Flink Operator with Volcano enabled
+
+Please refer to [Deploy the operator to a Kubernetes cluster](./user_guide.md#deploy-the-operator-to-a-kubernetes-cluster)
+
+# Create a sample Flink session cluster with batch scheduling enabled
```

Comment:
Sample session cluster and job cluster feel duplicate, maybe just introduce job cluster in detail, then briefly mention that "you can create a sample session cluster as well with kubectl apply -f config/samples/flinkoperator_v1beta1_flinksessioncluster_volcano.yaml".

Reply:
ok